test/reconciler.test.jsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx h */
+
+import { render } from "../src/reconciler"
+import { h } from "../src/h";
+
+const testRender = jsx => new Promise(resolve => {
+  let target = document.createElement("div")
+
+  document.body.appendChild(target)
+
+  render(jsx, target)
+
+  // TODO add callback to render function, e.g.:
+  //   render(jsx, target, () => resolve(target.innerHTML))
+  //   remove timeout hack below:
+  setTimeout(() => resolve(target.innerHTML), 100)
```

Comment:
As you see, until `render()` gets a `callback` argument, we can't really test the reconciler at all.

For now, I can test *one* execution only - and only by using a fixed `100` msec timeout to make sure the render finishes.

Until there's a render-callback, we can't really test the reconciler.